core/src/main/java/hudson/model/listeners/SaveableListener.java

```diff
@@ -76,6 +81,10 @@ public void unregister() {
      * Fires the {@link #onChange} event.
      */
     public static void fireOnChange(Saveable o, XmlFile file) {
+        if (!Functions.isExtensionsAvailable() && !NOTIFY_DURING_STARTUP) {
+            LOGGER.info("Skipping notification for " + o + " in " + file + " as Jenkins is starting up or shutting down");
```

Comment:
```suggestion
            LOGGER.log(Level.INFO, "Skipping notification for {0} in {1} as Jenkins is starting up or shutting down", new Object[]{ o, file });
```
nit: use the interpolation version instead